cmd/main.go

```diff
@@ -358,6 +358,9 @@ func createSkrWebhookManager(mgr ctrl.Manager, skrContextFactory remote.SkrConte
 		SkrWebhookMemoryLimits: flagVar.WatcherResourceLimitsMemory,
 		RemoteSyncNamespace:    flagVar.RemoteSyncNamespace,
 	}
+	if flagVar.CertificateManagement == "gardener" {
+		setupLog.Info("[SETUP] Request to use Gardener for certificate management is not supported yet")
```

Comment:
This log doesn't make too much sense, we also have to change the context after the implementation done. As the AC suggest, just log gardener is enabled is sufficient, or just skip log entirely. In general, we should not relays on logs to observe which CertificateManagement to use, maybe we can consider creating metrics for this in the future.

Reply:
Agreed, it doesn't make much sense to just randomly log something when its going to be removed later on anyway. **So for now replacing the logging message with a simple TODO message.**
Another reason to do this is: in order to get access to the `logger`, the function declaration has to be modified, which is too much for a temporary placeholder.